### PR TITLE
feat: update readme

### PR DESCRIPTION
**Pull request by Intevel**
Some changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,7 @@
 - [Integration](#integration)
 - [Extensions](#extensions)
 - [Tools](#tools)
+- [Interesting Articles](#articles)
```
Comment (paescuj) [inline README.md:16]:
```suggestion
- [Articles](#articles)
```

File: README.md [hunk 2]
```diff
@@ -60,11 +62,23 @@
 - [Tags M2M](https://github.com/dimitrov-adrian/directus-extension-tags-m2m-interface) - M2M driven tags interface.
 - [Cloudinary Upload Interface](https://github.com/kevcomparadise/directus-vue3-cloudinary-upload) - Simple interface for upload/read cloudinary image.
 - [Sanitize HTML](https://github.com/licitdev/directus-extension-sanitize-html) - Sanitize HTML inputs to Directus.
+- [Directus LogSnag](https://github.com/Intevel/directus-logsnag) - Sending your events from Directus directly to your phone using LogSnag
```
Comment (paescuj) [inline README.md:65]:
```suggestion
- [Directus LogSnag](https://github.com/Intevel/directus-logsnag) - Sending your events from Directus directly to your phone using LogSnag.
```

File: README.md [hunk 3]
```diff
@@ -60,11 +62,23 @@
...
+## Articles
+
+- [How to backup Directus](https://learndirectus.com/how-to-backup-directus/)
+- [Link Directus to a SMTP Relay](https://learndirectus.com/link-directus-to-a-smtp-relay/)
+- [How to use OAuth 2.0 with Directus](https://learndirectus.com/using-oauth-with-directus/)
+- [How to work with Many to Many relationships (M2M) on Directus](https://medium.com/@bianperotti/how-i-made-a-many-to-many-relationship-on-directus-b158ff55de7e)
+- [Build a blog with Nuxt 3 & Directus](https://directus.io/guides/get-started-building-a-nuxt-3-blog-with-directus/)
+- [How to Manage Your Supabase Database with Directus](https://directus.io/guides/directus-plus-supabase/)
+
+More articles you can find at [LearnDirectus](https://learndirectus.com/) & [Directus Guides](https://directus.io/guides/)
```
Comment (paescuj) [inline README.md:80]:
```suggestion
- [Directus Guides (Official)](https://directus.io/guides/)
- [Learn Directus](https://learndirectus.com/)
- [How to Work With Many to Many Relationships (M2M) On Directus](https://medium.com/@bianperotti/how-i-made-a-many-to-many-relationship-on-directus-b158ff55de7e)

```

**Comment by paescuj** [comment]
Hey @Intevel! Thank you very much for your pull request! Just a few changes and we're ready to go 👍

**Comment by Intevel** [comment]
@paescuj Resolved, thanks for your feedback! ❤️

**Comment by paescuj** [comment]
Nice, thx! ❤️

**Pull request merged**